apps/frontend/src/app/pages/profile/profile-follow/profile-follow.component.ts

```diff
@@ -21,6 +21,7 @@ export class ProfileFollowComponent implements OnInit {
   localFollowStatus: Follow; // The follow object of the local user following target user
   targetFollowStatus: Follow; // The follow object of the target user following local user
   checked = false;
+  isLoggedInUser: boolean;
```

Comment:
Creating new variable and checking if user is local is a way there, but actually parent component - profile.component already has that. And it even has check for local user for exactly this component on line 65 in profile.component.html. So the issue there is that the variable isLocal in profile.component doesn't work, so I would suggest to fix it there, rather than creating another variable.

Reply:
Good catch, I didn't realize that's what `isLocal` was referring to. I've updated the PR to use this approach, I agree this logic makes more sense in the parent component.